common/build/eslint-config-fluid/minimal-deprecated.js

```diff
@@ -24,6 +24,33 @@ const permittedImports = [
 	"*/index.js",
 ];
 
+// Restricted import patterns for all code.
+const restrictedImportPaths = [
```

Comment:
why is this in "minimal-deprecated.js" ? The file name makes me thing we arn't supposed to use this file? I'm guessing its that everything extends this buit it shouldn't be used directly? A comment in the top of the file woudl help.